app/services/albums.js

```diff
@@ -0,0 +1,28 @@
+const axios = require('axios'),
+  constants = require('../constants'),
+  errors = require('../errors'),
+  logger = require('../logger'),
+  photosService = require('./photos');
+
+const { ALBUM_API_URL } = constants;
+
+exports.getAlbum = async id => {
+  const url = `${ALBUM_API_URL}/albums?id=${id}`;
+  logger.info(`Making a request to url ${url}`);
+  try {
+    const albumInfo = await axios.get(`${url}`);
+    if (!albumInfo.data.lenght) {
```

Comment:
typo right here, remember to always test all cases by hand on top of your automatic tests